scripts/src/tests/services/users.spec.ts

```diff
@@ -125,4 +129,43 @@ describe("api tests for /users", async () => {
 		expect(await userExists("another-app", user.appUserId)).toBeFalsy();
 		expect(await userExists(user.appId, "another-user-id")).toBeFalsy();
 	});
+
+	test("testSign", async () => {
+		const app = await createApp(generateId(IdPrefix.App));
+		const payload = { test: "test" };
+		const jwt = await signJwt(app.id, "subject", payload);
+		const res = await verify<{ test: string }, "test_subject">(jwt);
+		expect(res.payload.test).toEqual(payload.test);
+	});
+
+	test("testMalformedJWT", async () => {
+		let jwt;
```

Comment:
please annotate these two with types

Reply:
What type `payload` should be?

Reply:
you can go with `object`, as defined in the function itself